lessons/Projects/project-research-paper/langs/en-us/index.adoc

```diff
@@ -39,45 +45,53 @@
 
 === Overview
 
-In this capstone project, students respond to a chosen question about their dataset. They use the data cycle to ask questions, consider, analyze, and interpret their data. They apply
+In this capstone project, students respond to a chosen question about their dataset. They use the data cycle to ask questions, consider, analyze, and interpret their data.
 
 Prior to embarking on this project, students should complete @lesson-link{project-data-exploration}, where they gain familiarity with a dataset and consider worthwhile questions to investigate.
 
+We've provided templates for students to create a @starter-file{research-project, "slide deck"} and a @starter-file{research-paper, "written report".} The slide deck offers more scaffolding, preparing students to write  research papers if you choose.
+
 === Launch
 
 Data Science is the process of learning from data. Any learning that _you_ do from _your_ data needs to be motivated by a *good question* - one that will produce new insights and rich discussion. That means it can't be too vague, too broad, or too specific!
 
 [.lesson-instruction]
-- While working on @starter{project-data-exploration}, you honed in on a few questions worth further investigation.
-- In small groups, share and discuss the questions you've landed on. Let your peers share their gut reactions to your questions. Talk to your partners about advantages and disadvantages to pursuing various questions.
+- While working on your Data Exploration, you honed in on a few questions worth further investigation.
+- In small groups, share and discuss the questions that you found most interesting. Let your peers share their gut reactions to your questions, then discuss the advantages and disadvantages of pursuing each question.
 
-@teacher{Some students will likely be confident in their questions, while others will need to revise and rethink their questions. This discussion is intended to be an open conversation, allowing students the opportunity to pick the best question for them. Choosing an appropriate question is the first step toward a successful research project!}
+@teacher{Choosing an appropriate question is the first step toward a successful research project! Some students will likely be confident in their question(s), while others will need to revise and rethink their question(s). This discussion is intended to be an open conversation, allowing students the opportunity to pick the best question for them.}
 
 === Investigate
 
 [.lesson-instruction]
```

Comment:
so, if I were having my kids do the exploration project, I would sit them down and walk them through the requirements before they started messing with their slide decks at all. I think it would even be nice to have a slide deck to use as I walked kids through the exploration project.

As a result, I put this as @lesson-instruction (with the wrong formatting) because I wanted this info to be on a slide and not in the presenter notes. Curious to hear your thougghts?

Reply:
OK, so it shouldn't be hidden in the `@teacher` notes. But there's still no reason for this to be a `@lesson-instruction`. It feels like you've confusing "anything I might want to tell students" with `@lesson-instruction`.

Perhaps we should rename the directive to make its use more clear. Maybe `@students-do-this-now`?

Reply:
okay, I'm returning to this comment and noticing that yes, there is an immediate action for kids here.  Do we have a different idea of "immediate action"?

Reply:
okay, realizing that instead of changing this to @teacher or not offering any directive, I should just change the instructions, because I think this deserves to be highlighted. How about if I say, "Write down all of the dates that your teacher provides and record them on your calendar." or something like that?

and yeah, @students-do-this-now would make things clearer.